guide/popular-topics/errors.md

```diff
@@ -165,13 +167,11 @@ client.login('your-token-goes-here');
 client.users.fetch('myId').then(someInitFunction);
 ```
 
-### MessageEmbed field names may not be empty.
-
-This error originates from calling `MessageEmbed#addFields()` with a field object's `name` property as an empty string. If you would like the title to be empty for a reason, you should use a zero width space, which can be input as `\u200b`.
+### EmbedBuilder field names may not be empty.
 
-### MessageEmbed field values may not be empty.
+This error originates from calling `EmbedBuilder#addFields()` with a field object's `name` property as an empty string. If you would like the title to be empty for a reason, you should use a zero width space, which can be input as `\u200b`.
 
-In conjunction with the previous error, this error results from calling `MessageEmbed#addFields()` with a field object's `value` property as an empty string. You can use a zero-width space if you would like this blank.
+In conjunction with the previous error, this error results from calling `EmbedBuilder#addFields()` with a field object's `value` property as an empty string. You can use a zero-width space if you would like this blank.
```

Comment:
If this is to stay, the name of the error should be updated as it isn't the name of the error you're going to get alongside it being inconsistent with the other errors (as they are all the names of the error).

Reply:
Resolved